src/test/java/org/opensearch/security/authtoken/jwt/JwtVendorTest.java

```diff
@@ -63,18 +73,20 @@ public void testCreateJwt() throws Exception {
         Assert.assertNotNull(jwt.getClaim("iat"));
         Assert.assertNotNull(jwt.getClaim("exp"));
         Assert.assertEquals(expectedExp, jwt.getClaim("exp"));
+        Assert.assertEquals(expectedRoles, EncryptionDecryptionUtil.decrypt(claimsEncryptionKey, jwt.getClaim("roles").toString()));
```

Comment:
Do you think we should add an additional assertion in this test?

```
Assert.assertNotEquals(expectedRoles, jwt.getClaim("roles"));
```

Explicitly asserting that the claim in the payload is encrypted and not the value that original value.

Reply:
Good catch! I will add that.